jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1007,39 +995,34 @@ code_blockt java_bytecode_convert_methodt::convert_instructions(
     // a new maximal key
     assert(a_entry.first==--address_map.end());
 
+    const std::string statement = bytecode_info[i_it->bytecode].mnemonic;
+
     // clang-format off
-    if(i_it->statement != "goto" &&
-       i_it->statement != "return" &&
-       i_it->statement != patternt("?return") &&
-       i_it->statement != "athrow" &&
-       i_it->statement != "jsr" &&
-       i_it->statement != "jsr_w" &&
-       i_it->statement != "ret")
+    if(statement != "goto" &&
+       statement != "return" &&
+       statement != patternt("?return") &&
+       statement != "athrow" &&
+       statement != "jsr" &&
+       statement != "jsr_w" &&
+       statement != "ret")
     {
       // clang-format on
       instructionst::const_iterator next=i_it;
       if(++next!=instructions.end())
         a_entry.first->second.successors.push_back(next->address);
     }
 
-    if(i_it->statement=="athrow" ||
-       i_it->statement=="putfield" ||
-       i_it->statement=="getfield" ||
-       i_it->statement=="checkcast" ||
-       i_it->statement=="newarray" ||
-       i_it->statement=="anewarray" ||
-       i_it->statement=="idiv" ||
-       i_it->statement=="ldiv" ||
-       i_it->statement=="irem" ||
-       i_it->statement=="lrem" ||
-       i_it->statement==patternt("?astore") ||
-       i_it->statement==patternt("?aload") ||
-       i_it->statement=="invokestatic" ||
-       i_it->statement=="invokevirtual" ||
-       i_it->statement=="invokespecial" ||
-       i_it->statement=="invokeinterface" ||
-       (threading_support && (i_it->statement=="monitorenter" ||
-       i_it->statement=="monitorexit")))
+    if(
```

Comment:
Suggest killing clang-format here to keep the nicer one-cndition-per-line format

Reply:
done